justfile

```diff
@@ -144,16 +144,27 @@ check-examples:
   just run-garn go-http-backend check
   just run-garn monorepo check
 
-update-flakefiles:
+update-flakefiles check="false":
   #!/usr/bin/env bash
+  # set -euo pipefail
```

Comment:
Is making this a comment intentional?

Reply:
No it wasn't. Thanks for the hint. It's fixed now